### PR TITLE
VIH-8547 committing missing file from orginal brach.

### PR DESCRIPTION
**Pull request by khizarkhanu-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-8547

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by MichaelKeaneathmcts — DISMISSED** [review]

**Review by Anaia — APPROVED** [review]

**Review by FarronF — APPROVED** [review]

**Review by kevin-hmcts — APPROVED** [review]
LGTM

File: azure-pipelines.yml [hunk 1]
```diff
@@ -250,7 +250,7 @@ resources:
     - repository: templates
       type: github
       name: hmcts/azure-devops-templates
-      ref: refs/heads/master
+      ref: refs/heads/optional-dotnet-test-ci-run
```
Comment (PrasannaDommalapati) [inline azure-pipelines.yml:253]: Should this be master?

Comment (khizarkhanu-hmcts) [reply azure-pipelines.yml:253]: yes for now, we will revert it when cvp sandbox is up

**Review by Anaia — DISMISSED** [review]

**Review by MichaelKeaneathmcts — DISMISSED** [review]

**Review by ChrisPaulP — APPROVED** [review]

**Review by omio-ma — APPROVED** [review]

**Review by MichaelKeaneathmcts — APPROVED** [review]

**Review by krishnapottekkat-hmcts — APPROVED** [review]

**Pull request merged**